Source/Message.swift

```diff
@@ -23,36 +23,12 @@
 //
 
 import Foundation
-import Willow
-import XCTest
 
-class LoggerConfigurationTestCase: XCTestCase {
-    func testThatLoggerConfigurationCanCreatedPreConfiguredTimestampConfiguration() {
-        // Given
-        let logLevels: [LogLevel] = [.debug, .info, .event, .warn, .error]
+/// A Message is a detailed log entry with a name and a dictionary of associated attributes.
+public protocol Message {
```

Comment:
I can't help but think these should be `LogMessage` objects. Then you should also have `LogMessageModifier` objects to modify log messages. It also keeps the overall library naming consistent. Thoughts?